app/models/repository/scopes.rb

```diff
@@ -2,10 +2,36 @@ module Repository::Scopes
   extend ActiveSupport::Concern
 
   included do
-    equal_scope *%w(
-      path
-      state
-    )
-  end
+    equal_scope *(%w(path state))
```

Comment:
Lint/AmbiguousOperator: Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication.